components/Button.js

```diff
@@ -9,9 +9,13 @@
 import styled from 'styled-components';
 
 const Button = styled.button`
-  color: #fff;
+  color: ${props => props.theme.grays.white};
   border-radius: 4px;
-  background: linear-gradient(180deg, #5cbc80 0%, #359258 100%);
+  background: linear-gradient(
+    180deg,
+    ${props => props.theme.supports.greenPrimary} 0%,
+    #056d2c 100%
```

Comment:
Both these colors are referred to, under "gradient 2" in the styleguide. Should they be called `props.theme.gradients.gradient2.start` and `props.theme.gradients.gradient2.end` or something similar instead?